pkg/kev/manifest.go

```diff
@@ -189,6 +199,17 @@ func (m *Manifest) ReconcileConfig(envs ...string) (*Manifest, error) {
 	return m, nil
 }
 
+func validateExtensions(services Services) error {
+	for _, s := range services {
+		_, err := config.K8SCfgFromMap(s.Extensions)
+		if err != nil {
+			return errors.Wrapf(err, "unable to validation extensions for service %s", s.Name)
```

Comment:
How about `K8s configuration validation failed for service %s` instead?

Reply:
ahah yep I actually had fixed this 😛